docs/enhancements/multi-rhel-fips.md

```diff
@@ -0,0 +1,137 @@
+---
+title: fips-with-multiple-rhel-versions
+authors:
+  - "@carbonin"
+creation-date: 2024-05-07
+last-updated: 2024-05-08
+---
+
+# Support FIPS for installers built for different RHEL releases
+
+## Summary
+
+In order for an OpenShift cluster to be considered FIPS compliant the installer
+must be run on a system with FIPS compliant openssl libraries. This means using
```

Comment:
Yes, but that's not really relevant for this topic. Do you still want it added?

Reply:
Without fips mode enabled, the binary won't use openssl at all, so I think it's a relevant detail to include.

Reply:
The three parts of the problem are:

1. you must use the dynamically-linked binary (`openshift-baremetal-install`/`openshift-install-fips`) b/c sometimes it will need to do FIPS
2. if you use a dynamically linked library, you must use a RHEL 9 userspace from 4.16 and may use one for pre-4.16
3. but if you want to do FIPS pre-4.16 you must use a RHEL 8 userspace